BraveShared/BraveStrings.swift

```diff
@@ -585,6 +585,7 @@ extension Strings {
 
 extension Strings {
     public static let blockPopups = NSLocalizedString("BlockPopups", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block Popups", comment: "Setting to enable popup blocking")
+    public static let followUniversalLinks = NSLocalizedString("FollowUniversalLinks", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Allow universal links to open in external apps", comment: "Setting to follow universal links")
```

Comment:
This text is too cryptic for average user imo, or needs additional explanation in settings cell body

Reply:
cc @jamesmudgett @anthonypkeane

Reply:
Got confirmations that copy is OK here:

James - https://bravesoftware.slack.com/archives/G01F4JUNXAQ/p1613663957390700
Anthony - https://bravesoftware.slack.com/archives/G01F4JUNXAQ/p1613758058430400?thread_ts=1613751704.426100&cid=G01F4JUNXAQ